### PR TITLE
Add 2.6 release note

### PR DESCRIPTION
**Pull request by ananzh**
### Issues Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3447
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by joshuarrrr — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## Version 2.6.0 Release Notes
+
+### 🛡 Security
+
+- [CVE-2022-37599] Bump loader-utils from `2.0.3` to `2.0.4` ([#3031](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3031))
+- [CVE-2022-37603] Bump loader-utils from `2.0.3` to `2.0.4` ([#3031](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3031))
+- [CVE-2022-25860] Bump simple-git from `3.15.1` to `3.16.0` ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
+- [CVE-2022-25881] Resolve http-cache-semantics from `4.1.0` to `4.1.1` ([#3409](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3409))
+- [Security] Bump hapi/statehood from `7.0.3` to `7.0.4` ([#3411](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3411))
+- [CVE-2023-25166] Bump formula from `3.0.0` to `3.0.1` ([#3416](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3416)
```
Comment (joshuarrrr) [inline release-notes/opensearch-dashboards.release-notes-2.6.0.md:10]:
```suggestion
- [CVE-2023-25166] Bump formula from `3.0.0` to `3.0.1` ([#3416](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3416))
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- [CVE-2022-24999] Resolve qs from `6.5.3` to `6.11.0` ([#3450](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3450))
+- [CVE-2022-25758] Bump node-sass from `6.0.1` to `7.0.3` and sass-loader from `10.2.1` to `10.4.1` to bump scss-tokenizer from `0.2.3` to `0.4.3` ([#3455](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3455))
+- [CVE-2020-24025] Bump node-sass from `6.0.1` to `7.0.3` ([#3455](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3455))
+
+### 📈 Features/Enhancements
+
+- Add disablePrototypePoisoningProtection configuration to prevent JS client from erroring when cluster utilizes JS reserved words ([#2992](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2992))
+- [Multiple DataSource] Refactor test connection to support SigV4 auth type ([#3456](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3456))
+- Make build scripts find and use the latest version of Node.js that satisfies `engines.node` ([#3467](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3467))
```
[the remark this answers is not in the document]
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md:21]: will move to maintenance

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- [CVE-2020-36632] Bumps flat from `4.1.1` to `5.0.2` ([#3419](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3419))
+- [CVE-2023-25653] Bump node-jose from `2.1.1` to `2.2.0` ([#3445](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3445))
+- [CVE-2022-24999] Resolve qs from `6.5.3` to `6.11.0` ([#3450](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3450))
+- [CVE-2022-25758] Bump node-sass from `6.0.1` to `7.0.3` and sass-loader from `10.2.1` to `10.4.1` to bump scss-tokenizer from `0.2.3` to `0.4.3` ([#3455](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3455))
+- [CVE-2020-24025] Bump node-sass from `6.0.1` to `7.0.3` ([#3455](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3455))
+
+### 📈 Features/Enhancements
+
+- Add disablePrototypePoisoningProtection configuration to prevent JS client from erroring when cluster utilizes JS reserved words ([#2992](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2992))
+- [Multiple DataSource] Refactor test connection to support SigV4 auth type ([#3456](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3456))
```
[the remark this answers is not in the document]
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md:20]: Will add it

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- [Docs] Fix documentation link for date math ([#3207](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3207))
+
+### 🛠 Maintenance
+
+- Bump `re2` and `supertest` ([#3018](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3018))
+- Upgrade vega-tooltip to `0.30.0` to support custom tooltips ([#3359](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3359))
+- Allow relaxing the Node.js runtime version requirement ([3402](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3402))
+- Relax the Node.js requirement to `14.20.1` ([3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
+- Bump the version of Node.js installed by `nvm` to `14.21.3` ([3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
```
[the remark this answers is not in the document]
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md:42]: will add

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+### 📝 Documentation
+
+- [Docs] Fix documentation link for date math ([#3207](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3207))
+
+### 🛠 Maintenance
+
+- Bump `re2` and `supertest` ([#3018](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3018))
+- Upgrade vega-tooltip to `0.30.0` to support custom tooltips ([#3359](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3359))
+- Allow relaxing the Node.js runtime version requirement ([3402](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3402))
+- Relax the Node.js requirement to `14.20.1` ([3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
```
[the remark this answers is not in the document]
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md:41]: Thanks. Then our changelog might also require a fix

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### 🛡 Security
+
+- [CVE-2022-37599] Bump loader-utils from `2.0.3` to `2.0.4` ([#3031](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3031))
+- [CVE-2022-37603] Bump loader-utils from `2.0.3` to `2.0.4` ([#3031](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3031))
+- [CVE-2022-25860] Bump simple-git from `3.15.1` to `3.16.0` ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
+- [CVE-2022-25881] Resolve http-cache-semantics from `4.1.0` to `4.1.1` ([#3409](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3409))
+- [Security] Bump hapi/statehood from `7.0.3` to `7.0.4` ([#3411](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3411))
+- [CVE-2023-25166] Bump formula from `3.0.0` to `3.0.1` ([#3416](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3416)
+- [CVE-2020-36632] Bumps flat from `4.1.1` to `5.0.2` ([#3419](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3419))
```
[the remark this answers is not in the document]
Comment (ananzh) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md:11]: will fix

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (joshuarrrr) [reply release-notes/opensearch-dashboards.release-notes-2.6.0.md]: Yeah, we'll need to audit/update the CHANGELOG in a separate PR anyway.

File: release-notes/opensearch-dashboards.release-notes-2.6.0.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+- [Docs] Fix documentation link for date math ([#3207](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3207))
+
+### 🛠 Maintenance
+
+- Bump `re2` and `supertest` ([#3018](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3018))
+- Upgrade vega-tooltip to `0.30.0` to support custom tooltips ([#3359](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3359))
+- Allow relaxing the Node.js runtime version requirement ([#3402](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3402))
+- Relax the Node.js requirement to `14.20.1` ([#3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
+- Bump the version of Node.js installed by `nvm` to `14.21.3` ([#3463](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3463))
```
Comment (joshuarrrr) [inline release-notes/opensearch-dashboards.release-notes-2.6.0.md:42]: as per @AMoo-Miki we should remove these, as the backport didn't make it: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3486

```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]

**Review by joshuarrrr — APPROVED** [review]

**Review by kavilla — APPROVED** [review]

**Pull request merged**